cli/src/commands/device/remove.rs

```diff
@@ -17,7 +18,14 @@ pub async fn main(args: Args) -> anyhow::Result<()> {
         config_dir.display(),
     );
 
-    let device = load_device_file(&config_dir, device).await?;
+    // FIXME: https://github.com/Scille/parsec-cloud/issues/8591
```

Comment:
I don't get why the device deletion is related to the config

Reply:
I use the wrong link :x

Should be:

```suggestion
    // FIXME: https://github.com/Scille/parsec-cloud/issues/8604
```

We need the configuration for deleting the data associated to the device (we need the value of `data_base_dir`).

Currently both the client and CLI use default value